manifests/1.3.2/opensearch-1.3.2.yml

```diff
@@ -0,0 +1,15 @@
+---
+schema-version: '1.0'
+build:
+  name: OpenSearch
+  version: 1.3.2
+ci:
+  image:
+    name: opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028
```

Comment:
Needs to be updated to new image

Reply:
You can update the source of this , close PR, let github actions open another one later.